messages/rule-crashed.txt

```diff
@@ -0,0 +1,21 @@
+ESLint encountered an uncaught exception from rule "<%- ruleName %>".
+
+<% if (pluginName) { %>
+The rule in question comes from the plugin "eslint-plugin-<%- pluginName %>". You should open an issue on that plugin's issue tracker. Be sure to include your configuration, file source code, and the error message and stack trace.
+<% } else { %>
+This is a core rule, so this exception may indicate a bug in ESLint. Please open an issue in our issue tracker, using the following information in addition to your exact source code and configuration.
+<% } %>
+
+Useful information:
+
+    ESLint version: <%- eslintVersion %>
+
+<% if (stackTrace) { %>
+    Exception stack trace:
+    <%- stackTrace %>
+<% } %>
+
+    You can run this command to get your configuration:
+        eslint --print-config path/to/your/file.js
+
+Be sure that your bug report also clearly calls out any unusual configuration properties, including but not limited to which parser you are using.
```

Comment:
Obviously this should be cleaned up if I can send the parser metadata into the template.